prow/cmd/hook/main.go

```diff
@@ -44,18 +44,17 @@ import (
 )
 
 type options struct {
-	port int
```

Comment:
Why did we remove this? It might break ppl

Reply:
I won't since the `--port` flag still exists through the `githubeventserver.Options`